websites/P/Podurama/metadata.json

```diff
@@ -0,0 +1,28 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "358402930191106049",
+		"name": "ion606"
+	},
+	"service": "Podurama",
+	"description": {
+		"en": "Google Podcast presence, but with the podcast artwork"
+	},
+	"url": "podurama.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/CNkpXlB.png",
```

Comment:
[Logos](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo) should be 512x512px

Reply:
oh. I took the logo from the issue so I assumed it was good my bad